CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.10.2 (2021-09-17)
+
+- Fixed cell renderer change when validator status changes. [#143](https://github.com/blackbaud/skyux-ag-grid/pull/143) (Thanks @[blackbaud-jeremymorgan](https://github.com/blackbaud-jeremymorgan)!)
```

Comment:
I feel like we're missing some words here. I'm not sure, but here's my best guess of what we mean:
```suggestion
- Fixed the cell renderer to refresh when the validator status changes. [#143](https://github.com/blackbaud/skyux-ag-grid/pull/143) (Thanks @[blackbaud-jeremymorgan](https://github.com/blackbaud-jeremymorgan)!)
```